app/src/controllers/filter/selectors.js

```diff
@@ -20,14 +18,6 @@ export const activeFilterSelector = createSelector(
   (filters, filterId) => filters.find((filter) => filter.id === filterId),
 );
 
-export const unsavedFilterIdsSelector = createSelector(
-  launchFiltersSelector,
-  userFiltersSelector,
-  (filters, savedFilters) =>
-    filters
-      .map((filter) => {
-        const savedFilter = savedFilters.find((item) => item.id === filter.id);
-        return savedFilter && !isEqual(filter, savedFilter) ? savedFilter.id : null;
-      })
-      .filter(Boolean),
+export const unsavedFilterIdsSelector = createSelector(launchFiltersSelector, (filters) =>
```

Comment:
What do we get if a previously saved filter with a positive `id` is in edit mode with unsaved changes?

Reply:
Good point.